### ISSUE TITLE
Optimizers with different learning rate multipliers

### ISSUE THREAD

**Issue by Chriskuei** [NTMC-Community/MatchZoo#728]
Default keras optimizers have a global learning rate. But sometimes setting different learning rates for different layers may result in better performance.
